### DISCUSSION TITLE
yarn create next-app returns error, but create react-app works just fine

### DISCUSSION THREAD

**Discussion by HrushiMakode** [vercel/next.js#33654] [Help] [answered]
![YYfFY](https://user-images.githubusercontent.com/43725762/151035885-9731c9fc-3e8c-4a4a-a74b-e32049ba167f.png)

Can someone help me fix this? And please, explain why this error occurs so in the future I'll know how to deal with it.
Many thanks!

StackOverFlow => https://stackoverflow.com/questions/67692005/yarn-create-next-app-returns-error-but-create-react-app-works-just-fine

Same problem happening to me !! , Please Help

**Comment by guscsales** [comment, marked as answer]
have you tried to create the app in another folder? Try to create in some place inside the `C:\`, seems the sentence "Estevão Urbano" is generating an error on console

**Reply by HrushiMakode** [reply]
This is due to having space in username
